dashboard/frontend/components/StatusRibbon.js

```diff
@@ -0,0 +1,25 @@
+const actionType = {
+    active: ["Running", "green-400"],
+    pause: ["Paused", "yellow-300"],
+    abort: ["Aborted", "red-500"],
+    done: ["Done", "gray-500"],
```

Comment:
We generally don't use arrays to store properties. This can be:
```suggestion
    done: {
    	name: "Done",
    	className: "gray-500"
    },
```

This way you can add more properties in this without changing indexes. It also makes it easier to read the code that actually uses it. (`status.className` makes more sense that `status[1]`).